### PR TITLE
chore: Push docker images on all *-master branch updates

### PR DESCRIPTION
**Pull request by placer14**

### CODE REVIEW

**Comment by ognots** [comment]
you also have to adjust `scripts/push-docker-tags.sh `
see https://github.com/filecoin-project/sentinel-drone/blob/master/scripts/push-docker-tags.sh

**Comment by ognots** [comment]
circleci workflows also needs a tag filter
```            
tags:
  only: /.*/
```

File: .circleci/config.yml [hunk 1]
```diff
@@ -77,5 +77,6 @@ workflows:
               only:
                 - master
                 - main
+                - /.*-master$/
             tags:
               only: /^v[0-9].*/
```
Comment (ognots) [inline .circleci/config.yml:82]: change to /.*/

**Review by ognots — APPROVED** [review]

**Comment by placer14** [comment]
Unfortunate that `chore/docker-push-on-all-master` matches the new filter and attempted to push. It failed due to the slash in the branch name. We could further restrict the pattern to match only a single word before `-master`.

**Pull request merged**